src/cider/nrepl/middleware/inspect.clj

```diff
@@ -65,7 +65,10 @@
 (defn inspect-reply*
   [msg value]
   (let [config (select-keys msg [:page-size :max-atom-length :max-coll-size
-                                 :max-value-length :max-nested-depth])
+                                 :max-value-length :max-nested-depth :spacious])
+        config (if (contains? config :spacious)
+                 (update config :spacious #(if (= % "false") false %))
```

Comment:
Seems worthwhile to also handle "true"

(I'm aware that `"true"` also can act as `true` but that kind of impurity is better coerced asap in the edges of our logic)